Packs/Armis/Integrations/ArmisEventCollector/README.md

```diff
@@ -10,23 +10,26 @@ This integration was integrated and tested with API V.1.8 of Armis API.
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
     | Server URL | URL of the Armis instance the event collector should connect to. | True |
-    | API Secret Key | The authorization token used for authentication of the Armis API. | True |
-    | Number of events to fetch per type | The maximum number of events to fetch per event type. | False |
-    | Trust any certificate (not secure) | When selected, certificates are not checked. | False |
-    | Use system proxy settings | Runs the integration instance using the proxy server (HTTP or HTTPS) that you defined in the server configuration. | False |
-    | Event types to fetch | Define which event types to fetch. | True |
+    | API Secret Key | The API Secret Key allows you to programmatically integrate with the Armis ecosystem. | True |
+    | Maximum number of events per fetch | Alerts and activity events |  |
+    | Maximum number of device events per fetch | Devices events |  |
+    | Trust any certificate (not secure) |  |  |
+    | Use system proxy settings |  |  |
+    | Event types to fetch |  | True |
+    | Events Fetch Interval | Alerts and activity events. | False |
+    | Device Fetch Interval | Device events. | False |
 
 4. Click **Test** to validate the URLs, token, and connection.
 
 ## Commands
 
-You can execute these commands from an Cortex XSIAM incident War Room ,as part of an automation, or in a playbook.
+You can execute these commands from a Cortex XSIAM incident War Room ,as part of an automation, or in a playbook.
 After you successfully execute a command, a DBot message appears in the War Room with the command details.
 
 ### armis-get-events
 
 ***
-Manual command to fetch and display events.
+Manual command to fetch and display events. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to events duplication and exceeding the API request limitation.
```

Comment:
```suggestion
Manual command to fetch and display events. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to event duplication and exceeding the API request limitation.
```